pkg/pgmodel/ingestor.go

```diff
@@ -107,7 +108,7 @@ func (i *DBIngestor) Ingest(tts []prompb.TimeSeries) (uint64, error) {
 
 	rowsInserted, err := i.db.InsertNewData(newSeries, data)
 	if err == nil && int(rowsInserted) != totalRows {
-		return rowsInserted, fmt.Errorf("Failed to insert all the data! Expected: %d, Got: %d", totalRows, rowsInserted)
+		// return rowsInserted, fmt.Errorf("Failed to insert all the data! Expected: %d, Got: %d", totalRows, rowsInserted)
```

Comment:
why is this commented out? Can't we support this check now?

Reply:
nope! since we batch rows from different requests together, when we get a number of rows-inserted back, we have no way to no which request they belong to.